File: Python_Books/算法图解/grokking_algorithms_COPY/03_recursion/python/02_greet.py
def greet2(name):
    print("how are you, ", name, "?")

def bye():
    print("ok bye!")

def greet(name):
    print("hello, ", name, "!")
    greet2(name)
    print("getting ready to say bye...")
    bye()

greet("adit")

File: Python_Books/算法图解/grokking_algorithms_COPY/03_recursion/lua/02_greet.lua
function greet2(name)
    print("how are you, " .. name .. "?")
end

function bye()
    print("ok bye!")
end

function greet(name)
    print("hello, " .. name .. "!")
    greet2(name)
    print("getting ready to say bye...")
    bye()
end

greet("adit")